docs/advisors/checks/mongodb-swap-allocation.md

```diff
@@ -0,0 +1,39 @@
+# MongoDB Swap Allocation
+
+## Description
+This check warns if there is no swap memory allocated to your instance.
+
+## Resolution
+
+MongoDB performs best where swapping can be avoided or kept to a minimum since retrieving data from swap on disk will always be slower than accessing data in RAM. However, if the system hosting MongoDB runs out of RAM, swapping can prevent the Linux OOM Killer from terminating the mongod process.
```

Comment:
```suggestion
MongoDB performs best if swapping can be avoided or kept to a minimum since retrieving data from swap on disk will always be slower than accessing data in RAM. However, if the system hosting MongoDB runs out of RAM, swapping can prevent the Linux OOM (Out of memory) Killer from terminating the mongod process.
```